Userland/Libraries/LibGUI/AbstractThemePreview.cpp

```diff
@@ -158,4 +158,31 @@ void AbstractThemePreview::paint_event(GUI::PaintEvent& event)
     paint_preview(event);
 }
 
+void AbstractThemePreview::center_window_group_within(Span<Window> windows, Gfx::IntRect const& bounds)
+{
+    VERIFY(windows.size() >= 1);
+
+    auto to_frame_rect = [&](Gfx::IntRect rect) {
+        return Gfx::WindowTheme::current().frame_rect_for_window(
+            Gfx::WindowTheme::WindowType::Normal, rect, m_preview_palette, 0);
+    };
+
+    int leftmost_x_value = windows[0].rect.x();
+    int topmost_y_value = windows[0].rect.y();
+    auto combind_frame_rect = to_frame_rect(windows[0].rect);
+    for (auto& window : windows.slice(1)) {
+        if (window.rect.x() < leftmost_x_value)
+            leftmost_x_value = window.rect.x();
+        if (window.rect.y() < topmost_y_value)
+            topmost_y_value = window.rect.y();
+        combind_frame_rect = combind_frame_rect.united(to_frame_rect(window.rect));
+    }
+
+    combind_frame_rect.center_within(bounds);
+    auto frame_offset = to_frame_rect(Gfx::IntRect {}).location();
```

Comment:
just `{}` should be fine and seems to be a more common style across the codebase